### PR TITLE
Update few dependencies

### PR DESCRIPTION
**Pull request by arvindkrishnakumar-okta**
## Description
Bump up some dependencies related to security vulnerabilities seen in build.

## Category

- [ ] Bugfix
- [x] Enhancement
- [ ] New Feature
- [ ] Configuration Change
- [ ] Versioning Change
- [ ] Unit Test(s)
- [ ] Documentation

### CODE REVIEW

File: impl/src/main/resources/com/okta/authn/sdk/version.properties [hunk 1]
```diff
@@ -14,4 +14,4 @@
 # limitations under the License.
 #
 
-${project.version}
\ No newline at end of file
+okta-auth-java=${project.version}
```
Comment (bdemers) [inline impl/src/main/resources/com/okta/authn/sdk/version.properties:17]: Pull this one out into a different PR and make sure we have a test for it.
IIRC this file isn't used anymore, as it _should_ be in META-INF

Comment (arvindkrishnakumar-okta) [reply impl/src/main/resources/com/okta/authn/sdk/version.properties:17]: sure, will do

Comment (arvindkrishnakumar-okta) [reply impl/src/main/resources/com/okta/authn/sdk/version.properties]: dropped this change from this PR.

**Comment by arvindkrishnakumar-okta** [comment]
> Looks good, just pull the version.prop change out into a different PR

done, dropped that change from this PR.
